src/Apps/Artist/Server/redirect.ts

```diff
@@ -0,0 +1,18 @@
+export function redirectWithCanonicalParams({ req, res }) {
+  if (hasCanonicalSort(req.query)) {
+    // all good
+    return
+  }
+
+  // else remove the offending sort, causing fallback to default sort
+  delete req.query.sort
+
+  const sanitizedQuerystring = new URLSearchParams(req.query).toString()
```

Comment:
**suggestion**: use `stringify` from `qs` (like [here](https://github.com/artsy/force/blob/5431bf01c586708b019fe76d8b4d15c0284070cf/src/Apps/Artwork/Utils/createCollectUrl.tsx#L26-L34))

Reply:
Nowadays I prefer the Remix [philosophy](https://remix.run/docs/en/1.16.0/pages/philosophy#web-standards-http-and-html) of doing url munging [without external libraries](https://dev.to/nerdyman/replacing-query-string-with-native-urlsearchparams-4kdg), but since this is server-side code and the library is already in the repo, there's no harm in using `qs`. It's a little more readable too :+1:

Refactored: 62839a2a24496aa8f01dbcc866217a1aedd602c4

Reply:
thanks!